README

```diff
@@ -0,0 +1,30 @@
+usage: ./call-home.sh OPTIONS
+
+Collects telemetry information and sends it to a Percona server. The data collection
+may be disabled via setting an environment variable [PERCONA_TELEMETRY_DISABLE].
```

Comment:
```suggestion
may be disabled via setting an environment variable [PERCONA_TELEMETRY_DISABLE=1].
```

Reply:
Let's probably keep it simple. If the variable is set, we don't collect any data. If not, we do.

Reply:
It will be not clear if it still works if `PERCONA_TELEMETRY_DISABLE=0`, so as it is implemented now is in my opinion OK.

Reply:
I, as user, prefer to get an exact and clear instructions what and how shall be passed to script, program, etc in order to make it do smth.